src/main/java/games/strategy/triplea/ui/logic/RouteCalculator.java

```diff
@@ -86,73 +66,67 @@ public static Point getClosestPoint(final Point source, final List<Point> pool)
    * @return A List of all possible Points depending in map Properties
    *         size may vary
    */
-  public List<Point> getPossiblePoints(final Point point) {
-    final List<Point> result = new ArrayList<>();
+  public List<Point2D> getPossiblePoints(final Point2D point) {
```

Comment:
Couldn't this method be implemented using `getPossibleTranslations()`?

```java
    return getPossibleTranslations().stream()
        .map(t -> t.transform(point, null))
        .collect(Collectors.toList());
```

That would partially solve the test coverage issue I mention below.

Reply:
Good catch. I was thrown off by the second parameter